src/SFA.DAS.EmployerApprenticeshipsService.Web.UnitTests/Orchestrators/EmployerAccountTransactionOrchestratorTests/WhenIGetAccountTransactions.cs

```diff
@@ -117,22 +119,71 @@ public void Arrange()
             Assert.IsTrue(result.Data.AccountHasPreviousTransactions);
         }
 
-        private void SetupGetTransactionsResponse(int year = 0, int month = 0)
+        [Test]
+        public async Task ThenOnlyOneLevyTransactionShouldBeShownForSummary()
         {
+            //Arrange
+            var levyTransactions = new List<LevyDeclarationTransactionLine>
+            {
+                CreateLevyTransaction(new DateTime(2017,5,18), 200),
+                CreateLevyTransaction(new DateTime(2017,6,18), 300),
+                CreateLevyTransaction(new DateTime(2017,7,18), 500)
+            };
+
+            var transactions = new List<TransactionLine>();
+
+            transactions.AddRange(levyTransactions);
+            transactions.Add(new PaymentTransactionLine { Amount = 200, TransactionType = TransactionItemType.Payment });
+
+            SetupGetTransactionsResponse(2018, 2, transactions);
+
+            //Act
+            var result = await _orchestrator.GetAccountTransactions(HashedAccountId, default(int), default(int), ExternalUser);
+
+            var actualTransactions = result?.Data?.Model?.Data?.TransactionLines;
+
+            var levyDeclaration =
+                actualTransactions?.SingleOrDefault(t => t.TransactionType == TransactionItemType.Declaration);
+
+            //Assert
+            Assert.IsNotNull(actualTransactions);
+            Assert.IsNotNull(levyDeclaration);
+            Assert.IsFalse(string.IsNullOrWhiteSpace(levyDeclaration.Description));
+            Assert.AreEqual(2, actualTransactions.Count);
+            Assert.AreEqual(levyTransactions.Sum(t => t.Amount), actualTransactions.Single(t => t.TransactionType == TransactionItemType.Declaration).Amount);
+        }
```

Comment:
A single assert per test makes CI test failures dead easy to diagnose - if the test case contains many asserts (like here) then a failing CI test is harder to diagnose. Could this be mitigated by adding a message to the Asserts? Just thinking if CI fails we'll see the test that failed and a stack trace and a message like "expecting null" but we won't be able to tell which assert specifically failed.

Reply:
This is something I've always been confused at in this project. I used to write tests with a single assert but others didn't so move that that approach. i'll split this up anyway

Reply:
Personally I like one-test:one-assert approach because I've found it makes diagnosing CI failures easier and keeps the intention of each test crystal clear. Multiple asserts also tend to require more setup which makes it harder to tell what specific scenario is being tests. Appreciate not everyone's pov though. If we are going to use one-test:many-asserts then adding messages to assert makes it easier to tell which assert failed.